build/write-img-mac.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Prerequisites: Docker
+# Tested on MacOS 12.2.1
+# Author: behoyh
+
+# NOTE: EXPERIMENTAL. Use at your own risk!
+# Requires Dockerfile to have written .iso image first, eg.
+# docker run -it -v /dev:/dev -v $HOME/raspios-img/output/:/root/raspios-img/ --privileged pimeet
+# where '$HOME/raspios-img/output/' is the location of the image on the host.
+
+IMG_FILE="$HOME/raspios-img/output/2022-01-28-raspios-bullseye-arm64.img"
```

Comment:
FYI I updated this file name to avoid having to change all the scripts whenever we upgrade OS version. Take a look at the recent scripts to see the new name.